.storybook/preview-head.html

```diff
@@ -1,2 +1 @@
 <link href="https://fonts.googleapis.com/css?family=Lato:300,400,700" rel="stylesheet">
-<script src="//players.brightcove.net/5344802162001/rkcQq7gAe_default/index.min.js"></script>
```

Comment:
This lil guy? Don't we need this lil guy?

Reply:
nope